### DISCUSSION TITLE
Create new row automation

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#7506] [Help] [answered]
Hello everyone

I want to copy a row to a new table after a delay using an automation, but the relationship type columns are not filling

Relationship type columns are : Equipement - Nom

![Capture](https://user-images.githubusercontent.com/106697208/187407000-8235502b-51f1-4571-b684-b673112a0d67.PNG)

The same problem with App action automation

**Comment by R2bEEaton** [comment, marked as answer]
Hi @rafikApp,

I have found that a single relationship can be set in Automations via Handlebars using the `_id` of the row.

In your example, I'm guessing this would be:
```
{{ trigger.row.Equipement.0._id }}
{{ trigger.row.Nom.0._id }}
```

As you can see from my use of `.0`, this will only work if you have a single relationship in that field. I have not yet figured out how to add multiple relationship fields. I tried separating the IDs by commas, passing them as a list... nothing worked. Hopefully we can get a better idea if one of the main developers replies.

Good luck! Reply with any questions or mark this as answered if this worked for you.

**Comment by rafikApp** [comment]
Thank you @R2bEEaton it works

**Comment by rafikApp** [comment]
@R2bEEaton  i have another issue

I want to display only the value of the relationship column without the id, in order the generate a cutsomize pdf using your method

![Capture2](https://user-images.githubusercontent.com/106697208/187421965-4102f459-24f7-44b1-baf9-93f99f7ed920.PNG)
![Capture1](https://user-images.githubusercontent.com/106697208/187421974-8f0cc1e2-e45c-4c8e-aa3c-d418aabd2891.PNG)

**Reply by R2bEEaton** [reply]
Hey, very similar solution here:

```
{{ Form.Fields.Equipement.0.primaryDisplay }}
```

This will select the first (0th) _equipement_ and get the `primaryDisplay` field.

**Comment by rafikApp** [comment]
Thank you @R2bEEaton 
But i stil have a problem, when i select an element from relationship picker it doesn't display in the embed 

![Capture1](https://user-images.githubusercontent.com/106697208/187443709-31ffb907-1aee-4d09-b5e3-d631aec4121d.PNG)
![Capture2](https://user-images.githubusercontent.com/106697208/187443728-c3521da6-6857-4620-a473-203ba78e7053.PNG)

**Reply by rafikApp** [reply]
Hi @R2bEEaton Any suggetion ?
